docs/user_guide/building_blocks/producers.md

```diff
@@ -0,0 +1,176 @@
+# Production of columns
```

Comment:
I would prefer "Producers" as section title

Reply:
Since I am also talking about the ProduceColumns task further below I didn't think it was a good idea, but I can still restructure it if you want

Reply:
not changed after discussion on Friday